packages/schemas/alterations/next-1681267285-fix-get-started-passwordless-status.ts

```diff
@@ -0,0 +1,32 @@
+import { sql } from 'slonik';
+
+import type { AlterationScript } from '../lib/types/alteration.js';
+
+const adminConsoleConfigKey = 'adminConsole';
+
+const alteration: AlterationScript = {
+  up: async (pool) => {
+    const tenantIds = await pool.many<{ id: string }>(sql`select id from tenants`);
+
+    await Promise.all(
```

Comment:
nit: it would be better for debugging if run in sequence.

Reply:
Hmmm, why is execution sequence matter in this case?